sysutils/pyinfra/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                pyinfra
+version             1.4
+revision            0
+
+homepage            https://pyinfra.com
+
+description         pyinfra automates infrastructure super fast at massive \
+                    scale.
+
+long_description    {*}${description}  It can be used for ad-hoc command \
+                    execution, service deployment, configuration management \
+                    and more. Core design features include super fast \
+                    execution over thousands of hosts with predictable \
+                    performance, agentless execution against \
+                    SSH/Docker/subprocess/winrm hosts, extendable with any \
+                    Python package as configured & written in standard \
+                    Python, and integrated with Docker, Vagrant/Mech & \
+                    Ansible out of the box.
+
+platforms           darwin
+license             MIT
+categories          sysutils python
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+supported_archs     noarch
+
+checksums           rmd160  3c428307414bc86d06126ddb7aeb5fbbba007cdd \
+                    sha256  3cad5ca81b4d842091f39a6e24ea8117e9c2fcb1a4abfcfb254bf1617d87dca0 \
+                    size    166876
+
+python.default_version \
+                    39
+
+depends_build-append \
+                    port:py${python.version}-setuptools
+
+depends_lib-append  port:py${python.version}-click          \
```

Comment:
I've always wondered why by convention we list `install_requires` dependencies as library deps even though they're not required at build time. I normally leave them as runtime deps in my ports, which is what the [Gentoo docs](https://wiki.gentoo.org/wiki/Project:Python/Dependencies#Finding_dependencies) also suggests (unless there are tests present).

Out of curiosity, is there any particular reason why we do this? cc @reneeotten.

Reply:
This is true - changed to `depends_run`